core/deployment/src/main/java/io/quarkus/deployment/pkg/builditem/LegacyJarRequiredBuildItem.java

```diff
@@ -1,9 +0,0 @@
-package io.quarkus.deployment.pkg.builditem;
-
-import io.quarkus.builder.item.MultiBuildItem;
-
-/**
- * marker build item that extensions can use to force legacy jar creation
- */
-public final class LegacyJarRequiredBuildItem extends MultiBuildItem {
```

Comment:
I am not sure if should outright remove this and the uberjar build item, or have a deprecation phase. I totally get why they were removed and adding a deprecation phase will increase the complexity, but I am wondering if other extensions depend on them

Reply:
lambda and azure functions were the only thing producing them in quarkusio/quarkus.   You thinking externally?

Reply:
Exactly. @loicmathieu @gastaldi do you guys remember?

Reply:
Google Cloud Functions need and über JAR not a legacy JAR.
I don't understand the rational of thsis change, so I can't say anything (except it's less "declarative" than just publishing the item correponding to the needed package type).

As for deprecation or not, I suspect other extensions may use it so it may be a good idea to ask on the quarkus-dev mailing list. Anyway, there will be a clear upgrade path.

Reply:
I don't remember seeing any external extension using the legacy-jar packaging type

Reply:
> As for deprecation or not, I suspect other extensions may use it so it may be a good idea to ask on the quarkus-dev mailing list. Anyway, there will be a clear upgrade path.

+1